fake/fake-request-service.js

```diff
@@ -279,9 +285,9 @@ var details = (function () {
             }
         }
     });
-    
+
     // stream subscribers
-    streamMock.on(window.location.origin + '/glimpse/message-stream/', function() {
+    streamMock.on(window.location.origin + '/glimpse/message-stream/', function () {
         summaries.stream();
     });
 })();
```

Comment:
So, this was just the default formatting from vs code.  I can change it, but it raises the question of, if we're all using vs code & this is the default formatting rules, why are we using something different?